Formula/w/whisperkit-cli.rb

```diff
@@ -0,0 +1,38 @@
+class WhisperkitCli < Formula
+  desc "Swift native on-device speech recognition with Whisper for Apple Silicon"
+  homepage "https://github.com/argmaxinc/WhisperKit"
+  url "https://github.com/argmaxinc/WhisperKit.git",
+      tag:      "v0.4.0",
+      revision: "59cb8516c708e3e2f18198002600026b5a1135ca"
+  license "MIT"
+
+  depends_on xcode: ["15.0", :build]
+  depends_on arch: :arm64
+  depends_on :macos
+  depends_on macos: :ventura
+  uses_from_macos "swift"
+
+  def install
+    system "swift", "build", "-c", "release", "--product", "whisperkit-cli", "--disable-sandbox"
+    bin.install ".build/release/whisperkit-cli"
+  end
+
+  test do
+    mkdir_p "#{testpath}/tokenizer"
+    mkdir_p "#{testpath}/model"
+    whisperkit_command = [
+      "#{bin}/whisperkit-cli",
+      "transcribe",
+      "--model",
+      "tiny",
+      "--download-model-path",
+      "#{testpath}/model",
+      "--download-tokenizer-path",
+      "#{testpath}/tokenizer",
+      "--audio-path",
+      test_fixtures("test.mp3"),
+      "--verbose",
+    ].join(" ")
+    assert_includes shell_output(whisperkit_command), "Transcription:"
```

Comment:
```suggestion
    ]
    assert_includes shell_output(*whisperkit_command), "Transcription:"
```

I think this should work and it's preferable to pass these things as separate arguments when possible.

Reply:
looks like this is throwing an error
```
  An exception occurred within a child process:
    ArgumentError: wrong number of arguments (given 11, expected 1..2)
```

Reply:
without the `*` it's throwing 

```
 sh: [/opt/homebrew/Cellar/whisperkit-cli/0.4.0/bin/whisperkit-cli,: No such file or directory
```

Reply:
so I reverted to the original one @MikeMcQuaid